### PR TITLE
Draft: [FIX] account: clean correctly the cache

### PR DESCRIPTION
**Pull request by fmdl**
Before this commit the cache is cleaned only of some field. If there are other module with fields depending of those fields, there are not invalidate.

This commit fix that.

@qdp-odoo @rco-odoo 

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by fmdl** [comment]
OPW #2838911

File: addons/account/wizard/accrued_orders.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def _get_default_company(self):
     preview_data = fields.Text(compute='_compute_preview_data')
     display_amount = fields.Boolean(compute='_compute_display_amount')
 
-    @api.depends('date', 'amount')
+    @api.depends('date', 'amount', 'preview_data')
```
Comment (rco-odoo) [inline addons/account/wizard/accrued_orders.py:57]: In the compute method, shouldn't `self.preview_data` be `record.preview_data` instead?

**Review by rco-odoo — COMMENTED** [review]
Makes sense to me.
But is this an actual bug fix?
Do you have an issue to refer to?

**Comment by rco-odoo** [comment]
@fmdl if you are still interested to merge this PR, can you please add a test to reproduce the issue?

**Pull request closed without merging**